core/scale/scale_decoder_stream.cpp

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "scale/scale_decoder_stream.hpp"
+
+#include "macro/unreachable.hpp"
+#include "scale/outcome_throw.hpp"
+#include "scale/scale_error.hpp"
+#include "scale/types.hpp"
+
+namespace kagome::scale {
+  namespace {
+    CompactInteger decodeCompactInteger(ScaleDecoderStream &stream) {
+      auto first_byte = stream.nextByte();
+
+      const uint8_t flag = (first_byte)&0b00000011u;
```

Comment:
```suggestion
      const uint8_t flag = (first_byte) & 0b00000011u;
```

Reply:
ok, but clang-format doesn't pay attention to it.